diff --git a/node_modules/@expo/cli/build/src/start/platforms/android/adb.js b/node_modules/@expo/cli/build/src/start/platforms/android/adb.js
index afd67e3..68eff0e 100644
--- a/node_modules/@expo/cli/build/src/start/platforms/android/adb.js
+++ b/node_modules/@expo/cli/build/src/start/platforms/android/adb.js
@@ -95,7 +95,7 @@ async function isPackageInstalledAsync(device, androidPackage) {
     return false;
 }
 async function launchActivityAsync(device, { launchActivity  }) {
-    return openAsync(adbArgs(device.pid, "shell", "am", "start", "-a", "android.intent.action.RUN", // FLAG_ACTIVITY_SINGLE_TOP -- If set, the activity will not be launched if it is already running at the top of the history stack.
+    return openAsync(adbArgs(device.pid, "shell", "am", "start", // FLAG_ACTIVITY_SINGLE_TOP -- If set, the activity will not be launched if it is already running at the top of the history stack.
     "-f", "0x20000000", // Activity to open first: com.bacon.app/.MainActivity
     "-n", launchActivity));
 }
